cmd/prodme/main.go

```diff
@@ -57,8 +57,9 @@ type AppConfigFile struct {
 
 var (
 	Version        = "No version provided"
-	configFilename = flag.String("config", "config.yml", "The filename of the configuration")
+	configFilename = flag.String("config", "", "The filename of the configuration")
```

Comment:
Please make this an actual default which uses the HOME environment variable. Example code:
https://github.com/Symantec/Dominator/blob/master/lib/srpc/setupclient/api.go